JS/02/table/table.js

```diff
@@ -0,0 +1,215 @@
+const users = []
+
+const infoUser = () => {
+  const boxPopup = document.querySelector('.box-popup')
+  const fullName = boxPopup.querySelector('.full-name').value
+  const gender = boxPopup.querySelector('.gender').value
+  const age = boxPopup.querySelector('.age').value
+
+  const user = {
+    fullName: fullName,
+    gender: gender,
+    age: age
+  }
+  users.push(user)
+  render(users)
```

Comment:
- Each function should only do one thing, if function name is collectUserInfo then only collect user info, dont add user to the list or render here
- 
```suggestion
const collectUserInfo = () => {}
const addUser = () => {
  const newUser = collectUserInfo();
  users.push(user)
  render(users)
}
```